dbOperations/removeUser.js

```diff
@@ -47,9 +47,14 @@ async function removeUser(teamId, userId) {
       //Remove user from user table where UID == userID
       const userRef = doc(db, "teams", teamId, "users", userId);
 
-      await transaction.delete(userRef);
+      const userSnapshot = await transaction.get(userRef);
 
-      await removePfp(getPfpName(teamId, userId));
+      //remove pfp if there is one
+      if (userSnapshot.data().pfp !== "") {
+        await removePfp(getPfpName(teamId, userId));
```

Comment:
This made me realize that we currently lack functionality that allows users to delete their existing profile pictures and revert to the default profile picture. I will work on creating a ticket for this.